.github/workflows/prs_create-cherry-pick-pr.yml

```diff
@@ -67,6 +73,7 @@ jobs:
         with:
           # the action will run for each value in matrix.branch
           branch: ${{ matrix.branch }}
+          token: ${{ secrets.token }}
```

Comment:
I didn't test it, it's a guess that it would work based on https://github.com/carloscastrojumo/github-cherry-pick-action?tab=readme-ov-file#action-inputs.

Reply:
yes, we can give it a try ... looks as if this might do the trick

Reply:
I think the fix for [this issue](https://github.com/mui/mui-x/pull/16531#discussion_r1950640822) should be here

Something like:

```diff
 - name: Cherry pick and create the new PR
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         uses: carloscastrojumo/github-cherry-pick-action@503773289f4a459069c832dc628826685b75b4b3 # v1.0.10
         with:
           # the action will run for each value in matrix.branch
           branch: ${{ matrix.branch }}
+          token: ${{ secrets.token || secrets.GITHUB_TOKEN }}
```